.gitignore

```diff
@@ -6,6 +6,21 @@ __pycache__/
 # C extensions
 *.so
 
+# Swap
+[._]*.s[a-v][a-z]
+[._]*.sw[a-p]
+[._]s[a-v][a-z]
```

Comment:
Shouldn't we just limit to `swp`, `swo` and `*~`? With these changes some other formats might be ignored too even though we don't use them! e.g. [some of these](https://en.wikipedia.org/wiki/List_of_filename_extensions_(S%E2%80%93Z)), I am approving it but just food for thought.  :angel:

Reply:
I took it from here: https://github.com/github/gitignore/blob/master/Global/Vim.gitignore